marinerescue-frontend/src/pages/ParentsPage.js

```diff
@@ -0,0 +1,108 @@
+                            <div class="column">
+                                <div class="content">
+                                    <img id="icon" src="images/safety-education-outcomes.png"></img>
+                                    <h2>Safety</h2><br />
+                                    <h3>Students will learn about marine debris, where it comes from, and its impact on marine life to encourage responsible citizenship development.
+
+This learning module includes an interactive story for students to learn about key marine debris concept, and an activity to practice characterizing different kinds of marine debris. </h3>
+                                </div>                            
+                            </div>
+                            <div class="column">
+                                <div class="content">
+                                    <img id="icon" src="images/ecosystems-education-outcomes.png"></img>
+                                    <h2>Ecosystems</h2><br />
+                                    <h3>Students will learn about marine debris, where it comes from, and its impact on marine life to encourage responsible citizenship development.
+
+This learning module includes an interactive story for students to learn about key marine debris concept, and an activity to practice characterizing different kinds of marine debris. </h3>
+                                </div>                            
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="section">
+                    <div class="footer">
+                    <div class="row">
+                            <div class="column">                        
+                            </div>
+                            <div class="column">
+                                <h3>Connect with us: </h3>
+                                <img id="icon" src="images/youtube-logo.png"></img> <br />
+                            </div>
+                            <div class="column">
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="column">
+                                <div class="navbar-logo">
+                                    <img src="images/logo.png"></img>
+                                </div>                            
+                            </div>
+                            <div class="column">
+                                <h3>@2021 AppleJACK</h3>
+                            </div>
+                            <div class="column">
+                                <img id="icon" src="images/ischool-logo.png"></img> <br />
+                                <img id="icon" src="images/coasst-logo.png"></img>
+
+                            </div>
+                        </div>
+                    </div>
+                </div>
```

Comment:
Can you please move this to be its own component <Footer> so we can re-use it everyone on our website? Thanks!